File: src/App.View/src/constants/rigths/crudOperation.js
export const NONE = { name: 'Не назначено', shortName: 'N', code: 0 }
export const CREATE = { name: 'Створення', shortName: 'C', code: 1 }
export const READ = { name: 'Редагування', shortName: 'R', code: 2 }
export const UPDATE = { name: 'Оновлення', shortName: 'U', code: 4 }
export const DELETE = { name: 'Видалення', shortName: 'D', code: 8 }
export const ALL = { name: 'Повний доступ', shortName: 'A', code: 16 }
export const BAN = { name: 'Заборона на буль-яку дію', shortName: 'B', code: 32 }

export const CRUD_OPERATION = {
  NONE,
  CREATE,
  READ,
  UPDATE,
  DELETE,
  ALL,
  BAN,
}

export const CRUD_OPERATION_LIST = [NONE, CREATE, READ, UPDATE, DELETE, ALL, BAN]

File: submodules/app.core/src/Core.Base/Administration/Enums/CrudOperation.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Core.Base.Administration
{
    [Flags]
    public enum CrudOperation
    {
        /// <summary>
        /// Nothing is selected. Means not initialized.
        /// </summary>
        None = 0,
        /// <summary>
        /// Create
        /// </summary>
        C = 1,
        /// <summary>
        /// Read
        /// </summary>
        R = 2,
        /// <summary>
        /// Update
        /// </summary>
        U = 4,
        /// <summary>
        /// Delete
        /// </summary>
        D = 8,
        /// <summary>
        /// All
        /// </summary>
        A = 16,
        /// <summary>
        /// Ban
        /// </summary>
        B = 32
    }
}
